sphinx-docs/Relationships.md

```diff
@@ -0,0 +1,50 @@
+# Relationships
+
+Many CALDERA abilities require input variables called "facts" to be provided before the ability can be run. These facts can either be provided manually through the CALDERA server GUI, or they can be discovered by a previous ability. 
```

Comment:
`These facts can be provided through fact sources, or they can be discovered by a previous ability.` will cover everything for the second sentence